core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Layout.java

```diff
@@ -56,7 +56,9 @@ public LabelCanvas draw(Driver driver, Barcodable barcodable) {
     @Override
     public LabelCanvas draw(Driver driver, Barcodable barcodable) {
       LabelCanvas label = driver.start(14, 4.666);
-      label.textClipped(10.5, 2.7, 0.5, 12, barcodable.getAlias());
+      // This label can handle 10 char/line, but we restrict it to 8 so that the barcodes break in a nice place for projects that people
+      // care about.
+      label.multilineText(10.7, 2.3, 0.5, 8, 2, Stream.of(new Pair<>(FontStyle.REGULAR, barcodable.getAlias())));
```

Comment:
This prints as 
```
PCSI_088
0_Pa_P...
```
Is that what's intended? I would have expected that breaking it at the 9th char would be friendlier, leading to
```
PCSI_0880
_Pa_P...
```